### DISCUSSION TITLE
Error when submit application

### DISCUSSION THREAD

**Discussion by N3EMESIS** [devchallenges-io/curriculum#3] [Report a bug]
?? You don't have enough to continue the screenshots to continue. Please upgrade to Pro or wait until 
![Captura de pantalla 2024-10-01 152055](https://github.com/user-attachments/assets/cad47119-5efd-42a9-b5cd-2835f2f338e3)
next month.

That's the message I get when I want to upload a challenge.

**Comment by nghiemthu** [comment]
Hi, you have reach limit of solution submission each month (Maximum 10 screenshots, each solution takes at least 1 screenshot). You can either wait for next month or upgrade to Pro to continue.

**Reply by N3EMESIS** [reply]
But I didn't do any challenge, I started with the first one and I got that error.

**Reply by nghiemthu** [reply]
@N3EMESIS I see that you are now able to submit the solution, did something change? was it related to the translation?
